build.sbt

```diff
@@ -65,13 +67,14 @@ lazy val magnoliaDependencies =
 
 lazy val refinedDependencies =
   libraryDependencies ++= {
-    if (scalaBinaryVersion.value == "2.11") Seq.empty // Just to make IntelliJ happy
+    if (scalaVersion.value == scala211.value) Seq.empty // Just to make IntelliJ happy
```

Comment:
This can happen several times. One setting can be `addDependency` which takes simply takes a list of scala versions to be excluded. Otherwise, different projects may have different ways to handle dependency settings. 

Also `refinedDependencies` is an unnecessary redirection, that it could be inlined as well

Reply:
That's a good idea, but it is the first time I saw such a case. I'm doing the same for other ecosystem projects. If I saw that the common case, I'll make a general utility inside zio-sbt.

Reply:
Just wrote these utility functions:

```scala
def addDependenciesFor(scalaBinaryVersions: String*)(dependencies: ModuleID*) =
  libraryDependencies ++= {
    if (scalaBinaryVersions.contains(scalaBinaryVersion.value)) dependencies else Seq.empty
  }

def addDependenciesExceptFor(scalaBinaryVersions: String*)(dependencies: ModuleID*) =
  libraryDependencies ++= {
    if (scalaBinaryVersions.contains(scalaBinaryVersion.value)) Seq.empty else dependencies
  }

def addScalacOptionsFor(scalaBinaryVersions: String*)(options: String*) =
  scalacOptions ++= {
    if (scalaBinaryVersions.contains(scalaBinaryVersion.value)) options else Seq.empty
  }

def addScalacOptionsExceptFor(scalaBinaryVersions: String*)(options: String*) =
  scalacOptions ++= {
    if (scalaBinaryVersions.contains(scalaBinaryVersion.value)) Seq.empty else options
  }
```